src/ocm/components/hosts/HostsDiscoveryTable.tsx

```diff
@@ -74,6 +89,35 @@ const HostsDiscoveryTable: React.FC<HostsDiscoveryTableProps> = ({
     [onEditHost, actionChecks.canEditHostname, actionChecks.canEditRole, onEditRole, cluster],
   );
 
+  React.useEffect(() => {
+    const forceRole = async () => {
+      if (cluster.hosts && cluster.hosts.length <= 3) {
+        try {
+          const promises = [];
+          for await (const host of cluster.hosts) {
```

Comment:
```suggestion
          for (const host of cluster.hosts) {
```
my bad no need for `await` here